### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sul-devops-team**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@jcoyne can you have a look at https://github.com/sul-dlss/common-accessioning/pull/671/commits/e09a1695c030516a188f85678dac5257c9d94b25#diff-2b588b7e60ae1c65e909888a03185725R39 in particular?

When I change that, rspec does this:

```shell
Randomized with seed 35438                                                                                                                            
.....................................................FF.....................                                                                          
                                                                                                                                                      
Failures:                                                                                                                                             
                                                                                                                                                      
  1) Robots::DorRepo::Accession::TechnicalMetadata.perform when a DRO with files when call to techmd service fails raises                             
     Failure/Error: expect { perform }.to raise_error(/Technical-metadata-service returned 500/)                                                      
       expected /Technical-metadata-service returned 500/ but nothing was raised                                                                      
     # ./spec/robots/accession/technical_metadata_spec.rb:100:in `block (5 levels) in <top (required)>'                                               
                                                                                                                                                      
  2) Robots::DorRepo::Accession::TechnicalMetadata.perform when a DRO with files when call to techmd service succeeds invokes techmd service          
     Failure/Error: expect(perform.status).to eq('noop')                                                                                              
                                                                                                                                                      
       expected: "noop"                                                                                                                               
            got: "skipped"                                                                                                                            
                                                                                                                                                      
       (compared using ==)                                                                                                                            
     # ./spec/robots/accession/technical_metadata_spec.rb:82:in `block (5 levels) in <top (required)>'                                                
                                                                                                                                                      
Finished in 0.85613 seconds (files took 1.46 seconds to load)                                                                                         
76 examples, 2 failures                                                                                                                               
                                                                                                                                                      
Failed examples:                                                                                                                                      
                                                                                                                                                      
rspec ./spec/robots/accession/technical_metadata_spec.rb:99 # Robots::DorRepo::Accession::TechnicalMetadata.perform when a DRO with files when call to
 techmd service fails raises                                                                                                                          
rspec ./spec/robots/accession/technical_metadata_spec.rb:81 # Robots::DorRepo::Accession::TechnicalMetadata.perform when a DRO with files when call to
 techmd service succeeds invokes techmd service         
```

**Comment by mjgiarlo** [comment]
@jcoyne thanks, made the change.

File: spec/robots/accession/technical_metadata_spec.rb [hunk 1]
```diff
@@ -47,6 +47,7 @@
                                         {
                                           externalIdentifier: '222-1',
                                           label: 'folder1PuSu/story1u.txt',
+                                          filename: 'folder1PuSu/story1u.txt',
```
Comment (mjgiarlo) [inline spec/robots/accession/technical_metadata_spec.rb:50]: @jcoyne is this what you'd expect here? is it OK if the filename has path info in it?

**Comment by mjgiarlo** [comment]
@jcoyne @ndushay worth another look. see too the question I dropped.

File: spec/robots/accession/technical_metadata_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/robots/accession/technical_metadata_spec.rb:50]: I think so

**Pull request merged**